### PR TITLE
Clean up stable structure migration - stop taking SchemaLabel from upgrade args

### PR DESCRIPTION
**Pull request by jasonz-dfinity**
# Motivation

Clean up stable structure migration code since the migration is completed.

# Changes

* Stop taking SchemaLabel from the upgrade args, and pretend it is always `AccountsInStableMemory`
* Refactor by replacement and remove dead code paths

# Tests

* Unit tests and local manual testing to make sure upgrade works

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: CHANGELOG-Nns-Dapp-unreleased.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ proposal is successful, the changes it released will be moved from this file to
 
 * Round neuron stake and maturity to 2 decimals in tables.
 * Show detailed stake amount in neuron details page.
+* Stop taking `schema_label` from upgrade arguments, which prevents triggering a data migration, and clean up related code.
```
Comment (dskloetd) [inline CHANGELOG-Nns-Dapp-unreleased.md:25]: I think it's probably not that useful to be very detailed about each step of the cleanup. If you write a more generic message here you don't have to add/update with each cleanup step.

File: rs/backend/src/accounts_store/schema/proxy/tests.rs
Comment (dskloetd) [inline rs/backend/src/accounts_store/schema/proxy/tests.rs:1]: The `proxy.rs` file still exists. Are you sure it should have zero tests?

**Review by dskloetd — APPROVED** [review]

File: CHANGELOG-Nns-Dapp-unreleased.md (hunk 1 above)
Comment (jasonz-dfinity) [reply CHANGELOG-Nns-Dapp-unreleased.md]: Changed to a more generic message.

File: rs/backend/src/accounts_store/schema/proxy/tests.rs
Comment (jasonz-dfinity) [reply rs/backend/src/accounts_store/schema/proxy/tests.rs:1]: Took another look. It seems the file now just contains some types and some trivial implementation with wrappers. Seems OK to leave them for the next clean up.

**Pull request merged**